cmd/meroxa/turbine/golang/deploy.go

```diff
@@ -104,6 +104,11 @@ func (t *turbineGoCLI) NeedsToBuild(ctx context.Context, appName string) (bool,
 	list, err := utils.GetTurbineResponseFromOutput(string(output))
 	if err == nil && list != "" {
 		// ignores any lines that are not intended to be part of the response
+		list = strings.Trim(list, "[]")
+		if list == "" {
+			return false, nil
+		}
+
 		hasFunctions := len(strings.Split(list, ",")) > 0
 		return hasFunctions, nil
 	}
```

Comment:
I think the code below this if statement should probably also be dropped. I'm not sure when it would be used? Right now it's only if list is empty or `GetTurbineResponseFromOutput` responded with an error.

Reply:
the code below is for older apps. we aren't requiring that people upgrade their apps, so apps that are older than a few days/a few weeks will need the regex below.  there wasn't the turbine-response tag before.